src/olympia/versions/tests/test_models.py

```diff
@@ -600,7 +601,7 @@ def test_version_deleted_not_synced_to_basket(
         new_version.delete()
         assert sync_object_to_basket_mock.delay.call_count == 0
 
-    def test_can_be_disabled_and_deleted(self):
+    def test_promoted_can_be_disabled_and_deleted(self):
         addon = Addon.objects.get(id=3615)
         # A non-recommended addon can have it's versions disabled.
```

Comment:
I wonder if these comments should be updated to say "promoted" instead of "recommended", and whether it would be worth using different promoted groups in the test, as opposed to just recommended?

Reply:
it kinda needs to be the same group throughout the test to test all the edge cases.  I thought about changing it to another promoted group but it doesn't give us anything new in terms of test coverage.  We could repeat it x more times to cover all the groups?  My thinking was this test would continue to cover all the edge cases and the next test double check the basic with some different groups.